src/status_im2/common/scroll_page/view.cljs

```diff
@@ -131,16 +123,17 @@
        [:f> f-scroll-page-header @scroll-height height name page-nav-right-section-buttons
         logo sticky-header top-nav title-colum navigate-back? collapsed?]
        [rn/scroll-view
-        {:content-container-style         {:flex-grow 1}
-         :shows-vertical-scroll-indicator false
-         :scroll-event-throttle           16
-         :on-scroll                       (fn [^js event]
-                                            (reset! scroll-height (int
-                                                                   (oops/oget
-                                                                    event
-                                                                    "nativeEvent.contentOffset.y")))
-                                            (when on-scroll
-                                              (on-scroll @scroll-height)))}
+        {:content-container-style           {:flex-grow 1}
+         :content-inset-adjustment-behavior :never
```

Comment:
`:content-inset-adjustment-behavior :never` this is it

Reply:
cc @J-Son89

Reply:
thanks @flexsurfer :)  can you explain further of what exactly this does?

Reply:
inset was added on ios to scrollview content, that's why you had to add conditions for ios and android

Reply:
nice! thanks for sharing this! 🙌